integration-test/common/src/test/java/com/tencent/rss/test/MockedShuffleServerGrpcService.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.test;
+
+import com.google.common.util.concurrent.Uninterruptibles;
+import io.grpc.stub.StreamObserver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.tencent.rss.proto.RssProtos;
+import com.tencent.rss.server.ShuffleServer;
+import com.tencent.rss.server.ShuffleServerGrpcService;
+
+import java.util.concurrent.TimeUnit;
+
+public class MockedShuffleServerGrpcService extends ShuffleServerGrpcService {
```

Comment:
Better name

Reply:
Is there any recommended name?

Reply:
`TimeoutShuffleServerGrpcService` may be better? But I'm not sure.

Reply:
MockedShuffleServerGrpcService might be better than TimeoutShuffleServerGrpcService, because we can add more simulation more than "RPC timeout" in the future.

Reply:
OK.